packages/rocketchat-lib/client/lib/index.js

```diff
@@ -0,0 +1,18 @@
+/*
+	What is this file? Great question! To make Rocket.Chat more "modular"
+	and to make the "rocketchat:lib" package more of a core package
+	with the libraries, this index file contains the exported members
+	for the *client* pieces of code which does include the shared
```

Comment:
@graywolf336 does that not affect the server as well?

Reply:
It does, but that file is specific to the client. The server has another one dedicated to it.